src/current/v23.2/encryption.md

```diff
@@ -70,7 +70,9 @@ The report shows encryption status for all stores on the selected node, includin
 - Active data key information.
 - The fraction of files/bytes encrypted using the active data key.
 
-CockroachDB relies on [storage layer]({% link {{ page.version.version }}/architecture/storage-layer.md %}) compactions to write new files using the latest encryption key. It may take several days for all files to be replaced. Some files are only rewritten at startup, and some keep older copies around, requiring multiple restarts. You can force storage compaction with the `cockroach debug compact` command (the node must first be [stopped]({% link {{ page.version.version }}/node-shutdown.md %}#perform-node-shutdown)).
+CockroachDB relies on [storage layer compactions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#compaction) to write new files using the latest encryption key. It may take several days for all files to be replaced. Some files are only rewritten at startup, and some keep older copies around, requiring multiple restarts. You can force storage compaction with the `cockroach debug compact` command (the node must first be [stopped]({% link {{ page.version.version }}/node-shutdown.md %}#perform-node-shutdown)).
+
+In some cases, the fraction of files/bytes encrypted on the store may never reach 100%. This can happen because the point in time at which encryption is enabled, CockroachDB only encrypts _new_ data written to the filesystem. Because it relies entirely on storage layer compactions, there's no mechanism by which dormant on-disk data is encrypted.
```

Comment:
I believe the percentage we surface in the DB Console is also the percentage encrypted _with the current data key_, which rotates at the configured `rotation-period`. When a data key rotates, the percentage will drop down to zero and slowly climb up as data is compacted. I think we should include some kind of statement on that here too.

Reply:
Thanks for that add'l info, I've updated as follows. The rotation info seems like the more common case so I've put that first in the list. 

PTAL and if it looks good I'll add it to the docs for all the versions in this PR

> The fraction of files/bytes encrypted on the store may be less than 100% for the following reasons:
> 1. The percentage shown is the percentage encrypted with the _current_ data key, which rotates at the configured [`rotation-period`](#starting-a-node-with-encryption). When a data key rotates, the percentage will drop down to zero and slowly climb up as data is compacted.
> 1. In some cases, it may never reach 100%. This can happen because from the point in time at which encryption is enabled, CockroachDB only encrypts _new_ data written to the filesystem. Because it relies entirely on storage layer compactions, there's no mechanism by which dormant on-disk data is encrypted.

Reply:
hi @jbowens what do you think of the latest updates? hope it's closer to accurate based on your feedback

Reply:
Looks good! Sorry for the delay reviewing 😅